controller/book.js

```diff
@@ -19,15 +19,15 @@ exports.findAllBooks = async (req, res) => {
 
 exports.findBookById = async (req, res) => {
   const { bookId } = req.params;
-  crudHelper.findEntryById(bookId, res, "book");
+  crudHelper.findEntryById(bookId, res, 'book');
```

Comment:
You could remove the variable creation on line 21 and access the `bookId` directly as you are only referencing the variable once:
```suggestion
  crudHelper.findEntryById(req.params.bookId, res, 'book');
```

Could apply the same to `updateBook` and `deleteBook` methods too